.github/workflows/build-images-ci.yaml

```diff
@@ -104,7 +104,7 @@ jobs:
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:latest-race
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
           build-args: |
-            BASE_IMAGE=quay.io/cilium/cilium-runtime:28b5a8658b596d12d80b0e7dad3efc2e77ec2d65@sha256:fbf8eee141101fade247dbe94cf84ca3bdcd92b96108996c50859ab7edd607d0
+            BASE_IMAGE=quay.io/cilium/cilium-runtime:6e2e3f2606e478b1c9a404070e3b21f2f5496445
```

Comment:
Why were the SHA256 hashes removed?

Reply:
Could this be an issue with the update-runtime-image script?

Reply:
This is expected. The SHA256 hash will be updated once the image build action ran for a second time and `make update-runtime-image` is run again. We might need to clarify that bit in https://docs.cilium.io/en/latest/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images

Reply:
@tklauser So they will be updated before we merge this PR, right?

Reply:
@pchaigno Yes, they will need to updated before mergeing.